applications/dds_video/README.md

```diff
@@ -0,0 +1,96 @@
+# DDS Video Application
+
+The DDS Video application demonstrates how video frames can be written to or
+read from a DDS databus in order to provide flexible integration between
+Holoscan applications and other applications (using Holoscan or not) via DDS.
+
+The application can be run as either a publisher or as a subscriber. In either case,
+it will use the [VideoFrame](../../operators/dds/video/VideoFrame.idl) data topic
+registered by the `DDSVideoPublisherOp` or `DDSVideoSubscriberOp` operators in order
+to write or read the video frame data to/from the DDS databus, respectively.
+
+When run as a publisher, the source for the input video frames will come from an
+attached V4L2-compatible camera via the `V4L2VideoCaptureOp` operator.
+
+When run as a subscriber, the application will use Holoviz to render the received
+video frames to the display. In addition to the video stream, the subscriber
+application will also subscribe to the `Square`, `Circle`, and `Triangle` topics
+as used by the [RTI Shapes Demo](https://www.rti.com/free-trial/shapes-demo).
+Any shapes received by this subscriber will also be overlaid on top of the
+Holoviz output.
+
+![DDS Video Application Workflow](docs/workflow_dds_video_app.png)
+
+## Building the Application
+
+This application requires [RTI Connext](https://content.rti.com/l/983311/2024-04-30/pz1wms)
+be installed and configured with a valid RTI Connext license prior to use.
+
+To build the application, the `RTI_CONNEXT_DDS_DIR` CMake variable must point to
+the installation path for RTI Connext. This can be done automatically by setting
+the `NDDSHOME` environment variable to the RTI Connext installation directory
```

Comment:
Just wondering: what is the meaning of `NDDSHOME`? Why not `DDSHOME`?

Reply:
Not sure why the N is there, that's defined by RTI Connext though.